versions/entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+set -u
+
+# Make sure BUILD_DIR is set and default to website
+if [ -z ${BUILD_DIR+x} ]; then
+  echo "BUILD_DIR is not set, falling back to default"
+  export BUILD_DIR="website"
+else
```

Comment:
Maybe we could remove the `else` statement and always print what the value of `BUILD_DIR` is

Reply:
The intention of this `else` was to know if you choose a new path or you use the default path in case you got some error related to it

Reply:
I see. I made the suggestion because currently, if `BUILD_DIR` is not set, the user won't know whats the `default` value. 
```bash
if [ -z ${BUILD_DIR+x} ]; then
  echo "BUILD_DIR is not set, falling back to default"
  export BUILD_DIR="website"
fi

echo "BUILD_DIR is set to '$BUILD_DIR'"
```
With something like this, the user will either see their `BUILD_DIR` value or the default one (which is "website")